spec/spec.md

```diff
@@ -846,10 +846,37 @@ For context on the Vaccination object passed, see the W3C-CCG [Vaccination
 Vocabulary](https://w3c-ccg.github.io/vaccination-vocab/), from which the
 example is drawn.
 
+### Message 4 - Ack Presentation
+
+Once the verifier validates the presentation, it can send a
```

Comment:
```suggestion
Once the verifier validates the presentation, it MAY send a
```

Reply:
Thanks, [updated](https://github.com/decentralized-identity/waci-presentation-exchange/compare/89dacf6f4d58736ab339e3b5c79e354f6f0383ab..0430d3ba7b4f0999389244d0c9347742ab2666b6) the text.